src/main/java/com/hermesworld/ais/galapagos/events/GalapagosEventSink.java

```diff
@@ -44,8 +43,8 @@
 
     CompletableFuture<Void> handleApplicationOwnerRequestCanceled(ApplicationOwnerRequest request);
 
-    CompletableFuture<Void> handleAddTopicProducers(List<String> producers, TopicMetadata metadata);
+    CompletableFuture<Void> handleAddTopicProducer(TopicMetadata metadata, String producerId);
 
-    CompletableFuture<Void> handleRemoveProducerFromTopic(TopicMetadata metadata, String appDeleteId);
+    CompletableFuture<Void> handleRemoveProducer(TopicMetadata metadata, String producerId);
```

Comment:
Why `handleAddTopicProducer`, but here, only `handleRemoveProducer`? Should be symmetrical (e.g. `handleRemoveTopicProducer` here). 

Also, parameter should be `producerApplicationId`.